### ISSUE TITLE
Split up the e2e tests

### ISSUE THREAD

**Issue by blu3beri** [hyperledger/aries-framework-javascript#377]
Right now the e2e test is one long test that takes quite a bit of time (mine took longer than 60 seconds...). It would be smart to split it up, so it could run in parallel.

**Comment by TimoGlastra** [comment]
We could also improve by maybe registering the schema / cred def only once or something.
